docker/api/build.py

```diff
@@ -121,6 +121,7 @@ def build(self, path=None, tag=None, quiet=False, fileobj=None,
         remote = context = None
         headers = {}
         container_limits = container_limits or {}
+        buildargs = buildargs or {}
```

Comment:
Any special reason not to use the default parameter value in line 20?
Looks like a `buildargs={}` in line 20 would be cleaner.

Reply:
@ulyssessouza https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
Ahhh, OK!